fastapi_sqla/utils.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
Is this module actually `fastapi_sqla.rds_support`?

Reply:
Yes I renamed it. Make total sense.